app/views/catalog/_index_default.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="row">
-  <%#= render_document_partial document, 'thumbnail', document_counter: generic_file_counter %>
+  <%= render 'thumbnail_default', document: document, document_counter: document_counter %>
```

Comment:
It is. https://github.com/projectblacklight/blacklight/blob/master/app/views/catalog/_thumbnail_default.html.erb